db/people.yml

```diff
@@ -2039,3 +2039,219 @@ zzgong:
   service: []
   skills: []
   title: Dr.
+rdifrancesco:
+  active: false
+  aka:
+    - Remus DiFrancesco
+  avatar: https://static-exp1.licdn.com/sc/h/djzv59yelk5urv2ujlazfyvrk
+  bio: bio
+  education:
+    - begin_year: 1992
+      degree: PhD
+      institution: msu
+      end_year: 1999
+      end_month: 12
+      group: bg
+      advisor: sbillinge
+  employment:
+    - begin_year: 2006
+      begin_month: 01
+      position: Senior Support Consultant
+      organization: SAP
+    - begin_year: 1992
+      position: Research Associate
+      organization: msu
+      status: phd
+      end_year: 1999
+      end_month: 4
+      group: bg
+  name: Remo DiFrancesco
+  position: Senior Support Consultant
+ppeterson:
+  active: false
+  aka:
+    - Peter Peterson
+  avatar: https://static-exp1.licdn.com/sc/h/djzv59yelk5urv2ujlazfyvrk
+  bio: bio
+  education:
+    - begin_year: 1999
+      degree: PhD
+      institution: msu
+      end_year: 2001
+      end_month: 12
+      group: bg
+      advisor: sbillinge
+  employment:
+    - begin_year: 2003
+      begin_month: 01
+      position: Software Engineer
+      organization: Oak Ridge National Laboratory
+    - begin_year: 1999
+      position: Research Associate
+      organization: msu
+      status: phd
+      end_year: 2001
+      end_month: 12
+      group: bg
+  name: Peter Peterson
+  position: Software Engineer
+ebozin:
```

Comment:
ebozin is already in the db so this can be removed, though check in case there is anything here that is missing in the other entry